sw/device/lib/dif/dif_spi_device.c

```diff
@@ -109,198 +119,251 @@ dif_spi_device_result_t dif_spi_device_init(
                              },
                              tx_fifo_end);
 
-  spi->rx_fifo_base = rx_fifo_start;
-  spi->rx_fifo_len = config->rx_fifo_len;
-  spi->tx_fifo_base = tx_fifo_start;
-  spi->tx_fifo_len = config->tx_fifo_len;
+  spi->rx_fifo_len = config.rx_fifo_len;
+  spi->tx_fifo_len = config.tx_fifo_len;
 
-  // Now that we know that all function arguments are valid, we can abort
-  // all current transactions and begin reconfiguring the device.
-  dif_spi_device_result_t err;
-  err = dif_spi_device_abort(spi);
-  if (err != kDifSpiDeviceResultOk) {
-    return err;
-  }
-  err = dif_spi_device_irq_reset(spi);
-  if (err != kDifSpiDeviceResultOk) {
-    return err;
-  }
```

Comment:
Is this no longer required? Could you please explain why?

Reply:
`configure` is not supposed to perform a reset.

Reply:
I see, thanks.